hudi-client/src/test/java/org/apache/hudi/table/TestCleaner.java

```diff
@@ -128,10 +128,8 @@ private void insertFirstBigBatchForClientCleanerTest(HoodieWriteConfig cfg, Hood
 
     assertFalse(table.getCompletedCommitsTimeline().empty());
     String instantTime = table.getCompletedCommitsTimeline().getInstants().findFirst().get().getTimestamp();
+    // We no longer write empty cleaner plans when there are not enough commits present
```

Comment:
do we have enough tests that now test when a valid cleaner plan?

Reply:
Yes, this particular check is when inserting first batch of data. There are other checks present which validates the effect of valid clean operations